tools-core/src/main/java/com/hedera/hashgraph/client/core/fileservices/FileAdapterFactory.java

```diff
@@ -16,23 +16,6 @@
  * limitations under the License.
  */
 
-/*
- * (c) 2016-2020 Swirlds, Inc.
- *
- * This software is the confidential and proprietary information of
- * Swirlds, Inc. ("Confidential Information"). You shall not
- * disclose such Confidential Information and shall use it only in
- * accordance with the terms of the license agreement you entered into
- * with Swirlds.
- *
- * SWIRLDS MAKES NO REPRESENTATIONS OR WARRANTIES ABOUT THE SUITABILITY OF
- * THE SOFTWARE, EITHER EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED
- * TO THE IMPLIED WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
- * PARTICULAR PURPOSE, OR NON-INFRINGEMENT. SWIRLDS SHALL NOT BE LIABLE FOR
- * ANY DAMAGES SUFFERED BY LICENSEE AS A RESULT OF USING, MODIFYING OR
- * DISTRIBUTING THIS SOFTWARE OR ITS DERIVATIVES.
- */
-
```

Comment:
Is there any reason to remove the license header?

Reply:
Yes, this is the wrong license: Originally the tools were part of the Swirlds repo and had a Swirlds license. Now they are part of the Hedera repo. 

For some reason Intellij did not remove all the old licenses when I did the change, but just appended them. I have removed 99% of the wrong files, but if I see one that wasn't removed, I do it manually.